project/henry0928/README.md

```diff
@@ -0,0 +1,113 @@
+# Voronoi Diagram and Delaunay Triangulation
+
+GitHub repository : https://github.com/henry0928/Voronoi-Diagram-and-Delaunay-Triangulation
+
+# Basic Information
+
+To put it simply, `Voronoi Diagram` is a kind of graphic segmentation. There will be many specific points on a planar graph. The collection of these specific points is called "A". All the points on the planar graph will go to "A" to find the specific point closest to itself, and form an independent area with the specific point. Then, the planar graph can be divided into various independent areas by using specific points to form a voronoi diagram.
+
+`Delaunay Triangulation`, A triangulation method invented by the mathematician Delaunay in 1934, that is, assuming that there is a set P formed by several points on the plane, then we can connect these points with lines, and any triangle formed by these line segments A circumscribed circle that does not have any other points inside it.
+
+In fact, `Delaunay Triangulation` and `Voronoi Diagram` are dual graphics.
+We can use Delaunay Triangulation to prodece Voronoi Diagram or use Voronoi Diagram to produce Delaunay Triangulation, either way will be possible.
+ 
+# Problem to Solve
+
+Both Voronoi Diagram and Delaunay Triangulation play an important role in the fields of facial recognition and geographic data analysis. We can use Voronoi Diagram to analyze the scope of responsibility of each fire brigade. We can also use Delaunay Triangulation to realize face transformation and facial features replacement.
+
+I'm trying to...
+1. Use `Fortune's Algorithm` to implement Voronoi Diagram and show the result graph with multiple points.
+2. Use Voronoi Diagram, which is created by us, to prodece Delaunay Triangulation on the same planar graph ( if time is enough ).
+
+# Fortune's Algorithm
```

Comment:
It's a good practice to describe the algorithm to be implemented in the proposal.